src/test/java/SignUp.java

```diff
@@ -65,13 +67,19 @@ public static void SignUpEmptyCredentials() throws InterruptedException {
         driver.get("https://deens.com/?internaltraffic");
         driver.manage().window().maximize();
 
-        driver.findElement(By.cssSelector("[href*='register']")).click();
+        driver.findElement(By.cssSelector("[href*='register']")).click();;
+        Thread.sleep(1000);
 
+        driver.findElement(By.cssSelector("[id=\'username\']")).sendKeys("");
+        driver.findElement(By.cssSelector("[id=\'email\']")).sendKeys("");
+        driver.findElement(By.cssSelector("[id=\'password\']")).sendKeys("");
         Thread.sleep(1000);
 
         driver.findElement(By.cssSelector("[class*=button]")).click();
+        Thread.sleep(1000);
 
         Assert.assertTrue(driver.findElement(By.cssSelector("[class=\'ui message\']")).isDisplayed());
```

Comment:
it would be better to get a text of message and compare it with expected text

Reply:
Once i register - there is no specific text of message i can use. 

I can only compare new elements that appear on registration page, such Avatar, images etc.

How should i proceed?

Reply:
ok